vite.config.ts

```diff
@@ -1,58 +1,53 @@
-import { NodeGlobalsPolyfillPlugin } from '@esbuild-plugins/node-globals-polyfill';
-import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfill';
-import rollupNodePolyFill from 'rollup-plugin-node-polyfills';
-import { sveltekit } from '@sveltejs/kit/vite';
-import { defineConfig } from 'vite';
-import { SvelteKitPWA } from '@vite-pwa/sveltekit';
-import { nostroidPWA } from './nostroid-config.js';
+import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
+import { NodeModulesPolyfillPlugin } from "@esbuild-plugins/node-modules-polyfill";
+import rollupNodePolyFill from "rollup-plugin-node-polyfills";
+import { sveltekit } from "@sveltejs/kit/vite";
+import { defineConfig } from "vite";
+import { SvelteKitPWA } from "@vite-pwa/sveltekit";
+import { nostroidPWA } from "./nostroid-config.js";
 
 export default defineConfig({
 	plugins: [
 		sveltekit(),
 		SvelteKitPWA(nostroidPWA),
 		NodeGlobalsPolyfillPlugin({
 			process: true,
-			buffer: true
+			buffer: true,
```

Comment:
While I understand that JS allowing these closing `,` is a good thing at times, please do not touch all the lines of files you touch prior to making this way of formatting stuff the project's standard. I don't want edit wars because the next dev's editor is set to remove those.

Reply:
My apologies, my editor had auto-formatting on. I created a .editorconfig to ensure that code style will be consistent across different editors. changed the formatting back to how it was